### ISSUE TITLE
Chocolatey

### ISSUE THREAD

**Issue by grxgghxrpxr** [digimezzo/dopamine-windows#1388]
Hello. I was wondering if it would be possible to update the package you've used on Chocolatey. The current version of Dopamine is 2.0.8 & will no longer download.

Thank you, I really appreciate it.

**Comment by ecogit** [comment]
Yes, [there](https://community.chocolatey.org/packages/dopamine) it's still 2.0.8 from June 2021.
